students/biegon_piotr/lesson_03_functions/input_validation.py

```diff
@@ -0,0 +1,21 @@
+print("Input Validation\n")
+
+
+def collatz(number):
+    if number % 2 == 0:
+        return number // 2
+    else:
+        return 3 * number + 1
+
+
+while 1:
+    try:
+        number = int(input("Enter a number: "))
+        if collatz(number) != 1:
+            print(collatz(number))
+        else:
+            break
+    except:
```

Comment:
it's too broad exception clause. We should not catch all exceptions, only those we can handle. In this scenario, we are looking for `ValueError` exception